aiida/orm/implementation/general/computer.py

```diff
@@ -513,6 +516,27 @@ def set_default_mpiprocs_per_machine(self, def_cpus_per_machine):
                 raise TypeError("def_cpus_per_machine must be an integer (or None)")
         self._set_property("default_mpiprocs_per_machine", def_cpus_per_machine)
 
+    def get_minimum_job_poll_interval(self):
```

Comment:
When I saw you used `metadata` as the container for this info, I was worried that this was useless, because as that is a column of the `DbComputer` table it should be immutable once stored, which would mean that it could only be set once during setup. I then checked the code and saw that the `is stored` check was commented out. So apparently, this is now possible, however, I wonder if it is the right thing. I don't think there is another viable option but I am just pointing it out here, so we can discuss and make sure this is what we want

Reply:
So I suspect this metadata is basically the _extras_ of Computer in the sense that they can be updated (intentionally) even after storing.